src/librustc_builtin_macros/asm.rs

```diff
@@ -264,27 +270,47 @@ fn parse_args<'a>(
     Ok(args)
 }
 
+fn err_duplicate_option<'a>(p: &mut Parser<'a>, span: Span) {
```

Comment:
Pass the `Symbol` for the option in as a parameter rather than recovering it from the span.